core/utils.py

```diff
@@ -1,23 +1,78 @@
 """Utility functions to load data from .mat .yaml and .h5 files."""
 
 import numpy as np
+from jax import numpy as jnp
 import h5py
 import os
 import yaml
 import scipy.io as spio
 import pickle
 from typing import Text
+from pynwb import NWBFile, NWBHDF5IO
+from ndx_pose import PoseEstimationSeries, PoseEstimation
 
+# Constants 
+MM_TO_M = 0.001
 
-def loadmat(filename):
+def daance_to_stac_mjx(data):
     """
```

Comment:
First line of the comment should be a 1-line (<88 chars total) short description of the function.

Reply:
Fixing.